### PR TITLE
Fix for missing thumbnails on advanced editor

### PR DESCRIPTION
**Pull request by igormuba**
Fix for the issue
https://github.com/ecency/ecency-vision/issues/1147

Just need to comment back into the code the line that outputs the images from the post

### CODE REVIEW

**Comment by igormuba** [comment]
Result of the change, allowing thumbnails to be seen:
image

**Review by feruzm — CHANGES_REQUESTED** [review]
Right, uncommenting fixes the issue. Reason I commented that out however was due to addition of thumbnails into json_metadata of the post object to blockchain which is unnecessary. We can just re-arrange images array and not include extra thumbnails array into posting json. So it needs improvements...

File: src/common/pages/submit.tsx [hunk 1]
```diff
@@ -826,7 +826,7 @@ class SubmitPage extends BaseComponent<Props, State> {
 
   buildMetadata = () => {
     const { tags, body, description, selectedThumbnail, selectionTouched } = this.state;
-    const meta = extractMetaData(body);
+    const { ["thumbnails"]: thumbnails, ...meta } = extractMetaData(body);
```
Comment (feruzm) [inline src/common/pages/submit.tsx:829]: @igormuba 
`const { thumbnails, ...meta} = extractMetaData(body)` should work as well, right?

Comment (igormuba) [reply src/common/pages/submit.tsx]: Yes, both have the same effect. Just to be sure I tried your way and it worked so I changed it, looks cleaner than my original solution, has the same effect

**Review by feruzm — APPROVED** [review]
lgtm

**Pull request merged**